src/components/Post/Post.types.ts

```diff
@@ -0,0 +1,6 @@
+interface Post {
+  id: string;
+  text: string;
+}
+
+export default Post;
```

Comment:
we have a convention for component props interfaces.
you place the interface inside the component file, there's usually no reason to have a separate file for types.
the name of the interface will be the name of the component, with a `Props` suffix.
also, we place the interface at the top of the file, above the component function.
e.g:
component name is `Post`
its props interface will be named: `PostProps`

```typescript
interface PostProps {
  id: string;
  text: string;
}

export function Post() {
  return <div>...</div>
}
```

Reply:
But I am using the type in more than one location